### ISSUE TITLE
All interactive elements are actionable when navigating without a mouse.

### ISSUE THREAD

**Issue by justpie** [Purdue/purdue-wp-theme#148]
The previous and next slide elements in the hero carousel are not accessible to keyboard and screen-reader users; they are currently plain text contained within span tags. Code this as a button type="button" with an aria-label, such as "previous slide" or "next slide", so keyboard and screen-reader users can access it and be aware of its function.
